tests/lsstseries_tests/test_ensemble.py

```diff
@@ -5,98 +5,91 @@
 
-@pytest.fixture
-def parquet_data():
-    ens = ensemble()
-    ens.from_parquet("tests/lsstseries_tests/data/test_subset.parquet", id_col='ps1_objid', band_col='filterName',
-                     flux_col='psFlux', err_col='psFluxErr')
 
-    yield ens
-    ens.client.close()
-
-
-def test_from_parquet(parquet_data):
+def test_from_parquet(parquet_ensemble):
     """
     Test that ensemble.from_parquet() successfully loads a parquet file
     """
-    ens = parquet_data
-
-    assert ens.data is not None  # Check to make sure the data property was actually set
-    ens.data = ens.data.compute()
-    for col in [ens._time_col, ens._flux_col, ens._err_col, ens._band_col]:
-        ens.data[col]  # Check to make sure the critical quantity labels are bound to real columns
-
-
-def test_counts(parquet_data):
+    # Check to make sure the data property was actually set
+    assert parquet_ensemble.data is not None
+     
+    parquet_ensemble.data = parquet_ensemble.data.compute()
+    for col in [
+        parquet_ensemble._time_col,
+        parquet_ensemble._flux_col,
+        parquet_ensemble._err_col,
+        parquet_ensemble._band_col,
+    ]:
+        # Check to make sure the critical quantity labels are bound to real columns
+        assert parquet_ensemble.data[col] is not None
+
+
+def test_counts(parquet_ensemble):
     """
     Test that ensemble.count() runs and returns the first five values correctly
     """
-    ens = parquet_data
-
-    count = ens.count()
+    count = parquet_ensemble.count()
     assert list(count.values[0:5]) == [499, 343, 337, 195, 188]
 
 
-def test_prune(parquet_data):
+def test_prune(parquet_ensemble):
     """
     Test that ensemble.prune() appropriately filters the dataframe
     """
-    ens = parquet_data
-
     threshold = 10
-    ens.prune(threshold)
-    assert ens.count(ascending=False).values[0] >= threshold
+    parquet_ensemble.prune(threshold)
+    assert parquet_ensemble.count(ascending=False).values[0] >= threshold
 
 
-def test_batch(parquet_data):
+def test_batch(parquet_ensemble):
     """
     Test that ensemble.batch() returns the correct values of the first result
     """
-    ens = parquet_data
-    result = ens.prune(10).dropna(1).batch(calc_stetson_J, band_to_calc=None)
+    result = (
+        parquet_ensemble.prune(10).dropna(1).batch(calc_stetson_J, band_to_calc=None)
+    )
 
-    assert pytest.approx(result.values[0]['g'], 0.001) == -0.04174282
-    assert pytest.approx(result.values[0]['r'], 0.001) == 0.6075282
+    assert pytest.approx(result.values[0]["g"], 0.001) == -0.04174282
+    assert pytest.approx(result.values[0]["r"], 0.001) == 0.6075282
 
 
-def test_to_timeseries(parquet_data):
+def test_to_timeseries(parquet_ensemble):
     """
     Test that ensemble.to_timeseries() runs and assigns the correct metadata
     """
-    ens = parquet_data
-    ts = ens.to_timeseries('88480000290704349')
+    timeseries = parquet_ensemble.to_timeseries(88480000290704349)
```

Comment:
similarly here, regarding naming, if we ever need to actually bring in the timeseries module this will conflict with the namespace